src/legacy/ui/ui_render/bootstrap/template.js.hbs

```diff
@@ -74,36 +67,27 @@ if (window.__kbnStrictCsp__ && window.__kbnCspNotEnforced__) {
       });
     }
 
-    load({
-      deps: [
+    load([
         {{#each sharedJsDepFilenames}}
           '{{../regularBundlePath}}/kbn-ui-shared-deps/{{this}}',
         {{/each}}
-      ],
-      urls: [
-        {
-          deps: [
-            '{{regularBundlePath}}/kbn-ui-shared-deps/{{sharedJsFilename}}',
-            {
-              deps: [
-                '{{dllBundlePath}}/vendors_runtime.bundle.dll.js'
-              ],
-              urls: [
-                {{#each dllJsChunks}}
-                  '{{this}}',
-                {{/each}}
-              ]
-            },
-            '{{regularBundlePath}}/commons.bundle.js',
-          ],
-          urls: [
-            '{{regularBundlePath}}/{{appId}}.bundle.js',
-            {{#each styleSheetPaths}}
-              '{{this}}',
-            {{/each}}
-          ]
-        }
-      ]
+        '{{regularBundlePath}}/kbn-ui-shared-deps/{{sharedJsFilename}}',
+        '{{dllBundlePath}}/vendors_runtime.bundle.dll.js',
+        {{#each dllJsChunks}}
+          '{{this}}',
+        {{/each}}
+        '{{regularBundlePath}}/commons.bundle.js',
+        {{!-- '{{regularBundlePath}}/plugin/data/data.plugin.js', --}}
+        '{{regularBundlePath}}/plugin/kibanaUtils/kibanaUtils.plugin.js',
+        '{{regularBundlePath}}/plugin/esUiShared/esUiShared.plugin.js',
+        '{{regularBundlePath}}/plugin/kibanaReact/kibanaReact.plugin.js'
+    ], function () {
+      load([
+        '{{regularBundlePath}}/{{appId}}.bundle.js',
```

Comment:
Execution order for plugins is not important and the script could be `async=true`. In practice, I don't see any difference even with x4 CPU throttling, though.

Reply:
For plugins? The execution order of the scripts which reference each other via the global `__kbnBundles__` variable definitely matters.